test/parallel/test-intl-no-icu-data.js

```diff
@@ -0,0 +1,10 @@
+// Flags: --icu-data-dir=test/fixtures/empty/
```

Comment:
Fair enough. I did a quick test and other tests are already failing without intl (see comment in main PR) so I'm okay with keeping the test as it is. If someone really does care about having clean runs with non-intl builds we can address in another PR.